uwmmss0348.xml

```diff
@@ -60,32 +60,36 @@
 				</extref>
 			</repository>
 			<physloc>wimiugma</physloc>
-			<abstract encodinganalog="520"
-				><!--This collection documents the activities of the Student Wisconsin Education Association. The majority of the documents are in scrapbook format, which highlights the 
-					group's outreach events such as Read Across America and Toys for Tots, as well as its board members. The collection also holds records for the Urban Teacher World  
-					Milwaukee program, which provides information about the program and their events for students. -->
-			</abstract>
+			<abstract encodinganalog="520">This collection documents the activities of the Student
+				Wisconsin Education Association. The majority of the documents are in scrapbook
+				format, which highlights the group's outreach events such as Read Across America and
+				Toys for Tots, as well as its board members. The collection also holds records for
+				the Urban Teacher World Milwaukee program, which provides information about the
+				program and their events for students. </abstract>
 			<langmaterial encodinganalog="546" label="Language">
 				<language encodinganalog="041" langcode="eng">English</language>
 			</langmaterial>
 		</did>
 		<bioghist encodinganalog="545">
-			<p><!--The documents in this collection date back to 2000, but there are mentions of the Student Wisconsin Education Association (SWEA) at UWM being founded sometime in the 
-				1990s. The organization is a branch of the National Education Association (NEA), founded in 1937, which assists student teachers in engaging in their communities and 
-				offers professional development opportunities. In the 2003-2004 academic year, SWEA was presented with two national awards: the Local Excellence Award, for its 
-				professional development opportunities and community outreach events; and the Outstanding CLASS Project Award, for their chapter book about group activities and their 
-				impact on the community.-->
-			</p>
+			<p>The documents in this collection date back to 2000, but there are mentions of the
```

Comment:
This needs to be a lot more formal. I did a quick google search and found a Student Org page (http://studentorgs.uwm.edu/show_profile/66208-student-wisconsin-education-association) and a Facebook page (https://www.facebook.com/UWMSWEA/) for the organization, which are two additional sources of information. Was hoping to find some mention on the UWM website as well, but no such luck. See: DACS 2.7.4